sudoku-mw/src/main/webapp/static/js/ie10-viewport-bug-workaround.js

```diff
@@ -7,6 +7,15 @@
 
 // See the Getting Started docs for more information:
 // http://getbootstrap.com/getting-started/#support-ie10-width
+$(document).ready(function() {
+	$("#hide").click(function() {
+		$("#new_game").hide();
+		});
+	$(window).load(function() {
+		$('#new_game').show();
+		$('#new_game').css("visibility", "visible");
+		});
+});
```

Comment:
This is external JS library for (as name implies for Internet Explorer  10 viewport bug workaround). Avoid modifying external files